### PR TITLE
Mpe invitations creator client

### PR DESCRIPTION
**Pull request by prastoin**
close https://github.com/AdonisEnProvence/MusicRoom/issues/349

### CODE REVIEW

File: packages/client/screens/MusicPlaylistEditorUsersSearchModal/__tests__/MpeRoomCreatorInviteUser.test.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    });
+
+    fireEvent.press(firstUserCard);
+
+    await waitFor(() => {
+        expect(serverSocketEventHandlerHasBeenCalled).toBeTruthy();
+    });
+});
+
+test('It should emit a creator invite user in mpe room event', async () => {
```
Comment (Devessier) [inline packages/client/screens/MusicPlaylistEditorUsersSearchModal/__tests__/MpeRoomCreatorInviteUser.test.tsx:59]: It seems that the test name is a duplication of the previous one.

File: packages/client/screens/UsersSearchEngineScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+interface UsersSearchEngineProps {
+    onUserCardPress: (userID: string) => void;
+}
+
+/**
+ * From now this "screen" is only used inside other wrapping other screen
+ * Calling it screen and leaving the file here because it's returning an AppScreenHeader
+ * Should this change ?
+ */
+const UsersSearchEngineScreen: React.FC<UsersSearchEngineProps> = ({
```
Comment (Devessier) [inline packages/client/screens/UsersSearchEngineScreen.tsx:59]: To me, we can keep it for now.

File: packages/client/screens/MusicPlaylistEditorRoomScreen/index.tsx [hunk 3]
```diff
@@ -276,6 +299,18 @@ const MusicPlaylistEditorRoomScreen: React.FC<MusicPlaylistEditorRoomScreenProps
...
                     }}
+                    HeaderRight={
+                        userIsMpeRoomCreator
+                            ? () => (
+                                  <InviteUserButton
+                                      testID={'mpe-invite-user-button'}
+                                      onInviteUser={
+                                          handleInviteUserButtonPressed
+                                      }
+                                  />
```
Comment (Devessier) [inline packages/client/screens/MusicPlaylistEditorRoomScreen/index.tsx:310]: The button breaks the UI of the header.

**Review by Devessier — CHANGES_REQUESTED** [review]
Great job!
I think there are two details to change before we can merge.

**Comment by prastoin** [comment]
Requested modifications have been done

**Comment by prastoin** [comment]
We might need to manage merge conflict after previous pending PR merging

**Comment by prastoin** [comment]
rebased on master

**Review by Devessier — APPROVED** [review]
Great job!
We can merge.

**Pull request merged**